frontend/cypress/support/nginx/Dockerfile.nginx.linux

```diff
@@ -0,0 +1,2 @@
+FROM nginx:1.21.1-alpine
```

Comment:
Why does the OS matter for Docker here?

Reply:
I've fixed the docker-compose and removed this.